pyqtgraph/graphicsItems/ArrowItem.py

```diff
@@ -17,7 +17,7 @@ def __init__(self, **opts):
         the setStyle() method.
         """
         self.opts = {}
-        QtGui.QGraphicsPathItem.__init__(self, opts.get('parent', None))
+        QtGui.QGraphicsPathItem.__init__(self, opts.pop('parent', None))
```

Comment:
Kind of a nitpick, but for consistency with other classes, could you make the `ArrowItem.__init__` signature `__init__(self, parent=None, **opts)` and use `parent` here instead?